packages/core/src/components/Dropdown/components/menu/menu.jsx

```diff
@@ -30,6 +30,7 @@ const Menu = ({ children, Renderer, selectProps, dropdownMenuWrapperClassName, i
   return (
     <components.Menu
       {...props}
+      innerProps={{ onScroll: props.onScroll }}
```

Comment:
can you just make sure that `props` do not already include `innerProps`? if it does, it would override it, so we'll need to spread it in like this:
```suggestion
      innerProps={{ ...props.innerProps, onScroll: props.onScroll }}
```

Do not commit it, check before if the `props` does include it

Reply:
it has innerProps, added

Reply:
cool, thanks 🙏🏼